src/tactic/dec_trivial.lean

```diff
@@ -0,0 +1,32 @@
+/-
+Copyright (c) 2020 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin
+-/
+import tactic.interactive
+
```

Comment:
Module doc please! Alternatively, I know we're generally moving away from the monolithic `tactic.interactive`, but since the copyright and module doc boilerplate will be longer than the content of this file, you could just stick the important bit in there.